_beacons/win_notify.py

```diff
@@ -171,13 +172,19 @@ def beacon(config):
                     confirm = _add_acl(path, mask, wtype, recurse)
                     sys_check = 1
                 if config[path].get('exclude', False):
-                    _remove_acl(path)
+                    for item in config[path]['exclude']:
+                        if '*' in exclude:
+                            for wildcard_item in glob.iglob(item):
+                                _remove_acl(wildcard_item)
+                        _remove_acl(item)
```

Comment:
Should this be under an `else`? You probably don't want to call `_remove_acl` on the path `C:\users\*\desktop`, right?